### DISCUSSION TITLE
Problema z2mqtt con coordinator Sonoff dongle-E

### DISCUSSION THREAD

**Discussion by Sacca36** [indomus/forum#2741] [HUB: Home Assistant]
Ciao a tutti, sto impazzendo da 2 giorni per cercare di installare z2mqtt con chiavetta sonoff dongle-e. attualmente sono con firmware NCP (ho provato anche RCP ma non si avvia comunque) e z2mqtt non si vuole avviare. Vi incollo anche il log ma non riesco a capire dove mettere le mani:

[10:28:30] INFO: Preparing to start...
[10:28:30] INFO: Socat not enabled
[10:28:33] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-12-03 10:28:40: Logging to console and directory: '/config/zigbee2mqtt/log/2023-12-03.10-28-40' filename: log.txt
Zigbee2MQTT:info  2023-12-03 10:28:40: Starting Zigbee2MQTT version 1.34.0 (commit #unknown)
Zigbee2MQTT:info  2023-12-03 10:28:40: Starting zigbee-herdsman (0.25.0)
Zigbee2MQTT:error 2023-12-03 10:29:01: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-12-03 10:29:01: Failed to start zigbee
Zigbee2MQTT:error 2023-12-03 10:29:01: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-12-03 10:29:01: Exiting...
Zigbee2MQTT:error 2023-12-03 10:29:01: Error: Connection not initialized
    at Ezsp.execCommand (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:551:19)
    at Ezsp.version (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:349:35)
    at Driver.startup (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/driver.ts:140:25)
    at EZSPAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/adapter/ezspAdapter.ts:172:16)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:60:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:107:5)

Qualcuno riesce ad aiutarmi?

**Comment by diaz66** [comment]
Il link stesso (Zigbee2MQTT:error 2023-12-03 10:29:01: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions) ti riporta una serie di verifiche possibili.
Se non ricordo male, mi capitò anche a me e risolsi collegando Sonoff dongle-E ad altra porta USB del raspberry.....

**Comment by KC-inDomus** [comment]
> Ciao a tutti, sto impazzendo da 2 giorni per cercare di installare z2mqtt con chiavetta sonoff dongle-e. attualmente sono con firmware NCP (ho provato anche RCP ma non si avvia comunque) e z2mqtt non si vuole avviare. Vi incollo anche il log ma non riesco a capire dove mettere le mani:
> 
> [10:28:30] INFO: Preparing to start... [10:28:30] INFO: Socat not enabled [10:28:33] INFO: Starting Zigbee2MQTT... Zigbee2MQTT:info 2023-12-03 10:28:40: Logging to console and directory: '/config/zigbee2mqtt/log/2023-12-03.10-28-40' filename: log.txt Zigbee2MQTT:info 2023-12-03 10:28:40: Starting Zigbee2MQTT version 1.34.0 (commit #unknown) Zigbee2MQTT:info 2023-12-03 10:28:40: Starting zigbee-herdsman (0.25.0) Zigbee2MQTT:error 2023-12-03 10:29:01: Error while starting zigbee-herdsman Zigbee2MQTT:error 2023-12-03 10:29:01: Failed to start zigbee Zigbee2MQTT:error 2023-12-03 10:29:01: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions Zigbee2MQTT:error 2023-12-03 10:29:01: Exiting... Zigbee2MQTT:error 2023-12-03 10:29:01: Error: Connection not initialized at Ezsp.execCommand (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:551:19) at Ezsp.version (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:349:35) at Driver.startup (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/driver.ts:140:25) at EZSPAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/adapter/ezspAdapter.ts:172:16) at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29) at Zigbee.start (/app/lib/zigbee.ts:60:27) at Controller.start (/app/lib/controller.ts:101:27) at start (/app/index.js:107:5)
> 
> Qualcuno riesce ad aiutarmi?

se non spieghi come hai istanziato zigbee2mqtt, che configurazione hai usato, come ti si aiuta? :)
